fuzzingandroid/start.bash

```diff
@@ -18,23 +18,92 @@ function wait_adb {
         adb disconnect $DEVICE > /dev/null 2>&1
         adb connect $DEVICE > /dev/null 2>&1
 
-        ! timeout 2 adb -s $DEVICE wait-for-device shell exit 0 > /dev/null 2>&1 || break
+        ! timeout 10 adb -s $DEVICE wait-for-device shell exit 0 > /dev/null 2>&1 || break
     done 
     
     adb -s $DEVICE wait-for-device
 }
 
 replace_monkey_uiautomator()
 {
-        adb -s $DEVICE push libs/monkey /sdcard
-        adb -s $DEVICE push libs/*.jar /sdcard
+	sleep 5
+	while true; do
```

Comment:
More system-level events? Adds a lot of downtime. Is this "hot", i.e., done often?